### PR TITLE
fix(icons): update icons again

### PR DESCRIPTION
**Pull request by tw15egan**
## Overview

Updates icons to final versions

### Added

- New icons
  - Button
  - Code Snippet
  - Content Switcher
  - Data Table
  - Modal
  - Notification
  - Search
  - Tooltip

### Changed

- Search close icon positioned based on 0 icons

### CODE REVIEW

**Review by marijohannessen — APPROVED** [review]
👍 ✅

**Review by alisonjoseph — CHANGES_REQUESTED** [review]
Looks good, but should be v9-rc as base branch right?

**Comment by marijohannessen** [comment]
@asudoh can we merge this into v9 and then cherry pick it to v9-rc?

File: src/components/search/_search.scss [hunk 1]
```diff
@@ -102,10 +102,6 @@
   }
 
   .#{$prefix}--search--lg .#{$prefix}--search-close {
-    right: rem(55px);
```
Comment (asudoh) [inline src/components/search/_search.scss:105]: I remember @aledavila brought this back recently. That said, is this change intentional…?

**Review by asudoh — APPROVED** [review]
LGTM basically 👍 - Thanks @tw15egan! @aledavila is right that this PR should be based on `v9-rc`, when doing so, please make sure the Git branch is off of `v9-rc`, too. Thanks!

File: src/components/search/_search.scss (hunk 1 above)
Comment (tw15egan) [reply src/components/search/_search.scss:105]: Yes, removing this as it was throwing off the positioning of the close icon. The close icon should always be `12px` from the right

**Comment by alisonjoseph** [comment]
Might be easier to create a new branch off of v9-rc and copy these changes over.
